robotbase/src/main/java/com/team2357/lib/util/Utility.java

```diff
@@ -56,6 +56,16 @@ public static WPI_TalonFX createDriveTalonFX(int CANID, double secondsToFull) {
     return talon;
   }
 
+  /**
+   * 
+   * @param CANID The CAN ID of the motor
+   */
+  public static WPI_TalonFX createShooterTalonFX(int CANID) {
+    WPI_TalonFX talon = new WPI_TalonFX(CANID);
+    talon.setNeutralMode(NeutralMode.Brake);
```

Comment:
Why are we setting brake on shooter rollers? They should coast.

Reply:
Shhhhhhh,